mr/src/main/java/org/apache/iceberg/mr/IcebergInputFormat.java

```diff
@@ -0,0 +1,494 @@
+/*
```

Comment:
Can I suggest putting this and the related classes into a `org.apache.iceberg.mr.mapreduce` package? So we can then add the `org.apache.iceberg.mr.mapred` implementation alongside it and the package names match the Hadoop API package names?

Reply:
@massdosage . I'll make this change once all other comments are addressed. I think making this change before can trip up github and it may consider mr.InputFormat and mr.mapreduce.InputFormat as two different files.

Reply:
OK, sure.

Reply:
Fixed.